Gems/Multiplayer/Code/Source/MultiplayerSystemComponent.cpp

```diff
@@ -853,7 +878,23 @@ namespace Multiplayer
         // Spawn the default player for this host since the host is also a player (not a dedicated server)
         if (m_agentType == MultiplayerAgentType::ClientServer)
         {
-            NetworkEntityHandle controlledEntity = SpawnDefaultPlayerPrefab(0);
+            IMultiplayerSpawner* spawner = AZ::Interface<IMultiplayerSpawner>::Get();
+            NetworkEntityHandle controlledEntity;
+            if (!spawner)
+            {
+                // If no spawner we have a default option
+                // make sure the player prefab path is lowercase (how it's stored in the cache folder)
+                auto sv_defaultPlayerSpawnAssetLowerCase = static_cast<AZ::CVarFixedString>(sv_defaultPlayerSpawnAsset);
```

Comment:
Same.. kill this cvar and codepath. We should *only* support the spawner interface system.